wit/golem-ts.wit

```diff
@@ -2,21 +2,4 @@ package golem:ts;
 
 world golem-ts{
```

Comment:
I think we wanted people to have access to everything we provide just by adding a dependency to our SDK - we have the same in golem-go and golem-rust. So they don't have to manually add wasi / golem imports.

Reply:
yes

Reply:
The types were / are not shipped with this thing anyway, so it wouldn't help users with that.
We could include the types, but I could imagine that leading to problems depending on how users want to use this.
(As there would be potentially multiple ambient declarations that users need to deal with).

Our template takes care of that properly as the types will be generated for everything that you are using in your wit.